docs/How-To-Create-Evergreen-Projects.md

```diff
@@ -0,0 +1,22 @@
+# How to Create Evergreen Projects
+
+This guide is meant to help users create their own projects on Evergreen without creating a ticket and waiting. 
+
+## Authorization
+
+To have access to create a project you must be either a super user or the admin of an existing project.
+If you do not see the `New Project` button on the [project settings page](https://spruce.mongodb.com/project/YourProject/settings/general), or do not have access to the project settings page, you may follow the steps under 'For Evergreen Users' in the wiki below.
+
+Note that that projects can only be created in spruce as it has been deprecated from the legacy UI.
+
+## Steps to Create
+
+Follow the steps on the [wiki](https://wiki.corp.mongodb.com/display/BUILD/How+to+Create+a+New+Evergreen+Project)
```

Comment:
Oh sorry I actually meant the opposite thing here -- the mongoDB wiki can link to this wiki. We should try to keep MongoDB links out of this repo since it's public. We might just want to avoid documenting any Evergreen ticket thing in the Evergreen wiki and keep it in the mongodb wiki since that's when it's actually relevant (same for line 22)

Reply:
updated